src/docs/product/integrations/splunk/index.mdx

```diff
@@ -94,10 +94,10 @@ After navigating to **Data Forwarding**, enable the Splunk integration:
 
 ![](splunk-data-forwarding-setting.png)
 
-Your instance URL is going to vary based on the type of Splunk service you’re using. If you’re using self-service Splunk Cloud, the instance URL will use the `input` prefix:
+Your instance URL is going to vary based on the type of Splunk service you’re using. If you’re using self-service Splunk Cloud, the instance URL will be <host> the url of the Splunk instance that runs HEC:
```

Comment:
this reads a little strangely to me, maybe something like:

"Your instance URL is going to vary based on the type of Splunk service you’re using. If you’re using self-service Splunk Cloud, the instance URL will be the URL of the Splunk instance running the HTTP Event Collector (HEC):"